rocksdb_admin/admin_handler.h

```diff
@@ -88,7 +94,8 @@ class AdminHandler : virtual public AdminSvIf {
   std::unique_ptr<ApplicationDBManager> db_manager_;
   RocksDBOptionsGeneratorType rocksdb_options_;
   // Lock to synchronize async_tm_backupDB(), async_tm_restoreDB(),
-  // async_tm_closeDB() and async_tm_changeDBRoleAndUpStream().
+  // async_tm_closeDB(), async_tm_changeDBRoleAndUpStream() and
+  // async_tm_setDBOptions()
   common::ObjectLock<std::string> db_admin_lock_;
 };
```

Comment:
Can you also add async_tm_clearDB() and async_tm_addS3SstFilesToDB() ?

Or just replace it with something like "Lock to synchronize DB admin operations at per DB granularity"